openedx/core/lib/block_structure/transformers.py

```diff
@@ -107,22 +107,22 @@ def transform(self, block_structure):
         collection, in the order that the transformers were added.
         """
         self._transform_with_filter(block_structure)
-        self._full_transform(block_structure)
+        self._transform_without_filter(block_structure)
```

Comment:
I could go for `_transform_with(out)_filters` though, to pluralize the object of the action.